### DISCUSSION TITLE
Need to apply mouseover event

### DISCUSSION THREAD

**Discussion by theDestI** [remotion-dev/remotion#1095] [Q&A]
Hi, for my use case I need to apply mouseover even on a dynamically loaded button inserted into a component via dangerouslySetInnerHTML. Would it be possible with remotion?

**Comment by JonnyBurger** [comment]
Yes, Remotion just renders plain React, so in principle, you can do whatever you can do in React as well.
You can look up React resources to find out to how to do it specifically!
